app/packages/looker-3d/src/fo3d/MediaTypeFo3d.tsx

```
@@ -376,6 +376,132 @@ export const MediaTypeFo3dComponent = () => {
+        const labelFieldData = sample.sample[field];
+
+        let thisLabel = null;
+
+        if (Array.isArray(labelFieldData)) {
+          // if the field data is an array of labels
+          thisLabel = labelFieldData.find(
+            (l) => l._id === labelId || l.id === labelId
+          );
+        } else if (
+          labelFieldData &&
+          labelFieldData.detections &&
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)